css/main.css

```diff
@@ -51,6 +52,62 @@ header {
   color: white;
 }
 
+/* Mobile Menu */
+
+.mb_menu {
+  background-color: #3c3a39;
+
+  /* position: absolute; */
+
+  /* z-index: 1; */
+  position: fixed;
+  z-index: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  display: none;
+  padding-right: 1px;
+}
+
+#x_icon {
+  padding: 15px 25px 10px 325px;
```

Comment:
- [ OPTIONAL ] I would advise against using fixed absolute units as it violates the principles of RWD